src/app/components/SplitButton.svelte

```diff
@@ -0,0 +1,51 @@
+<script lang="ts">
+  // props
+  export let colorLeft: string = "blue"; // optional
+  export let colorRight: string = "grey"; // optional
+  export let isOutlined: boolean = false; // optional
+  export let hasDropShadow: boolean = false; // optional
+  export let size: string = "medium"; // optional
+  export let leftText: string;
+  export let rightText: string;
+  export let data_cy_left: string;
+  export let data_cy_right: string;
+  export let onClickLeft: (ev: MouseEvent) => void;
+  export let onClickRight: (ev: MouseEvent) => void;
+</script>
+
+<style>
+  .split-button-layout {
+    display: inline-flex;
+    padding: 3%;
+  }
+  button {
+    margin: 0em;
+    border-radius: 0rem;
+    float: left;
+    width: 10rem;
+  }
+  button:first-child {
+    border-top-left-radius: 0.625em;
+    border-bottom-left-radius: 0.625em;
+  }
+  button:last-child {
+    border-left: 0em;
+    border-top-right-radius: 0.625em;
+    border-bottom-right-radius: 0.625em;
```

Comment:
I like this border rounding effect!

```suggestion
    border-top-left-radius: 0.625em;
    border-bottom-left-radius: 0.625em;
  }
  button:last-child {
    border-top-right-radius: 0.625em;
    border-bottom-right-radius: 0.625em;
```